modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionsExchangeFuture.java

```diff
@@ -394,6 +397,10 @@ public GridDhtPartitionsExchangeFuture(
             }
         };
 
+        MetricRegistry sysreg = cctx.kernalContext().metric().registry(SYS_METRICS);
```

Comment:
Can we pass metric via constructor or use `findMetric` here instead of trying to create it on each PME?